en/Obsidian Sync/Limitations.md

```diff
@@ -19,8 +19,10 @@ At the moment, each file can be up to 100 MB.
 ## How long do you keep my data after my subscription expires?
 
 We keep data in your remote vaults, including version history, for one month after your subscription expires.
+Your local vaults, stored on your devices, will not be impacted.
 
```

Comment:
```suggestion
In case you don't and your remote vaults no longer exist, you can easily [[Set up Obsidian Sync|create a new remote vault and re-connect your local vault]] again.
```